ValetTouchIDTest/ValetTouchIDTestViewController.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  ValetTouchIDTestViewController.swift
+//  Valet
+//
+//  Created by Eric Muller on 4/20/16.
+//  Copyright © 2016 Square, Inc. All rights reserved.
+//
+
+import UIKit
+
+
+final class ValetTouchIDTestViewController : UIViewController
+{
+    // MARK: Properties
+
+    @IBOutlet var textView : UITextView?
+    let secureEnclaveValet : VALSecureEnclaveValet = VALSecureEnclaveValet.init(identifier: "UserPresence", accessControl: VALAccessControl.UserPresence) as VALSecureEnclaveValet!
```

Comment:
ohhhhh I think I see what you're doing. Can't you just:
`let secureEnclaveValet = VALSecureEnclaveValet(identifier: "UserPresence", accessControl: VALAccessControl.UserPresence)`

I think your use of `.init` is throwing the compiler for a loop. Just a guess though